### PR TITLE
bump things and update documentation

### PR DESCRIPTION
**Pull request by Fogapod**
updated python to 3.11
added python setup guide to readme, comments to config files etc
merged deploy and docker build CI steps
added working mypy configuration (commented out because mypy is disabled for some reason)
removed all 57 build time dependencies
fixed a few lints

### CODE REVIEW

**Comment by Fogapod** [comment]
I did not test docker compose because i don't have it but docker image builds and seems to run fine

File: README.md [hunk 1]
```diff
@@ -1,40 +1,80 @@
...
 
-4- Test out the webui by accessing http://localhost:8000/
+You will need python 3.11+
 
-##### Navigating web UI
+#### Install venv (only first time or after updating sytem python version)
 
-Assuming you've managed to get a page running on http://localhost:8000/, we can now start doing things such as registering a test account.
+```sh
+python -m venv .venv
```
Comment (corp-0) [inline README.md:32]: Poetry will handle environments for you. It creates a new one per project.

Comment (Fogapod) [reply README.md:32]: I don't have it installed globally. For poetry projects i first create venv, then install and use poetry inside.

Could add both versions to readme but i don't know how to make venv with poetry properly

Comment (corp-0) [reply README.md:32]: poetry is intended to be installed locally. Poetry will create the venv for the project when you run ``poetry install``, there is nothing you have to do except that.

Comment (Fogapod) [reply README.md:32]: Hid venv stuff under spoiler. Rendered: https://github.com/unitystation/central-command/blob/01ba1d9e5477f91971f8a354f995b8889e6e4641/README.md

**Review by corp-0 — APPROVED** [review]

**Pull request merged**